src/main/resources/GoogleApps/TestDocumentList.xml

```diff
@@ -20,34 +20,22 @@
  * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
 -->
 
-<xwikidoc>
+<xwikidoc version="1.4" reference="GoogleApps.TestDocumentList" locale="">
   <web>GoogleApps</web>
   <name>TestDocumentList</name>
   <language/>
   <defaultLanguage/>
   <translation>0</translation>
-  <parent>GoogleApps.WebHome</parent>
   <creator>xwiki:XWiki.Admin</creator>
+  <parent>GoogleApps.WebHome</parent>
   <author>xwiki:XWiki.Admin</author>
-  <customClass/>
   <contentAuthor>xwiki:XWiki.Admin</contentAuthor>
   <version>1.1</version>
   <title>Document List</title>
-  <defaultTemplate/>
-  <validationScript/>
   <comment/>
   <minorEdit>false</minorEdit>
   <syntaxId>xwiki/2.0</syntaxId>
   <hidden>true</hidden>
-  <attachment>
-    <filename>testcolors.xls</filename>
-    <filesize>7168</filesize>
-    <mimetype>application/vnd.ms-excel</mimetype>
-    <author>xwiki:XWiki.Admin</author>
-    <version>1.1</version>
-    <comment/>
-    <content/>
-  </attachment>
```

Comment:
It seems the test attachment is lost. Can you check if it's on the file system and attached at build time?

Reply:
Indeed. At build time, it wasn't attached. So I added the code back.